modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/upgrade/v5_9_0/CommerceAccountUpgradeProcess.java

```diff
@@ -0,0 +1,201 @@
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.commerce.internal.upgrade.v5_9_0;
+
+import com.liferay.account.constants.AccountConstants;
+import com.liferay.account.model.AccountEntry;
+import com.liferay.account.service.AccountEntryLocalService;
+import com.liferay.commerce.util.CommerceAccountHelper;
+import com.liferay.expando.kernel.model.ExpandoTable;
+import com.liferay.expando.kernel.model.ExpandoValue;
+import com.liferay.expando.kernel.service.ExpandoTableLocalService;
+import com.liferay.expando.kernel.service.ExpandoValueLocalService;
+import com.liferay.portal.kernel.dao.orm.ActionableDynamicQuery;
+import com.liferay.portal.kernel.dao.orm.RestrictionsFactoryUtil;
+import com.liferay.portal.kernel.model.Group;
+import com.liferay.portal.kernel.model.TypedModel;
+import com.liferay.portal.kernel.service.ClassNameLocalService;
+import com.liferay.portal.kernel.service.GroupLocalService;
+import com.liferay.portal.kernel.service.ResourceLocalService;
+import com.liferay.portal.kernel.service.WorkflowDefinitionLinkLocalService;
+import com.liferay.portal.kernel.service.WorkflowInstanceLinkLocalService;
+import com.liferay.portal.kernel.upgrade.UpgradeProcess;
+import com.liferay.portal.kernel.workflow.WorkflowConstants;
+
+import java.sql.ResultSet;
+import java.sql.Statement;
+
+import java.util.function.UnaryOperator;
+
+/**
+ * @author Drew Brokke
+ */
+public class CommerceAccountUpgradeProcess extends UpgradeProcess {
+
+	public CommerceAccountUpgradeProcess(
+		AccountEntryLocalService accountEntryLocalService,
+		ClassNameLocalService classNameLocalService,
+		CommerceAccountHelper commerceAccountHelper,
+		ExpandoTableLocalService expandoTableLocalService,
+		ExpandoValueLocalService expandoValueLocalService,
+		GroupLocalService groupLocalService,
+		ResourceLocalService resourceLocalService,
+		WorkflowDefinitionLinkLocalService workflowDefinitionLinkLocalService,
+		WorkflowInstanceLinkLocalService workflowInstanceLinkLocalService) {
+
+		_accountEntryLocalService = accountEntryLocalService;
+		_classNameLocalService = classNameLocalService;
+		_commerceAccountHelper = commerceAccountHelper;
+		_expandoTableLocalService = expandoTableLocalService;
+		_expandoValueLocalService = expandoValueLocalService;
+		_groupLocalService = groupLocalService;
+		_resourceLocalService = resourceLocalService;
+		_workflowDefinitionLinkLocalService =
+			workflowDefinitionLinkLocalService;
+		_workflowInstanceLinkLocalService = workflowInstanceLinkLocalService;
+	}
+
+	@Override
+	protected void doUpgrade() throws Exception {
+		String selectCommerceAccountSQL =
+			"select * from CommerceAccount order by commerceAccountId asc";
```

Comment:
Probably this will fails cause the CommerceAccount table is no more.
Not sure we can move the UpgradeProcesses like this.